### PR TITLE
Create For Applicants.md

### PR DESCRIPTION
**Pull request by kmonterr**
Trying to create new pages for applicants under resources

Fixes issue(s) # .

[CircleCI](https://circleci.com/gh/18F/nsf-sbir/tree/BRANCH_NAME)

[:sunglasses: PREVIEW](https://federalist.fr.cloud.gov/preview/18f/nsf-sbir/BRANCH_NAME/)

[Preview README for this branch](https://github.com/18F/nsf-sbir/blob/BRANCH_NAME/README.md)

Changes proposed in this pull request:
-
-
-

/cc @relevant-people

### CODE REVIEW

**Comment by kmonterr** [comment]
[For applicants Page.docx](https://github.com/18F/nsf-sbir/files/1129255/For.applicants.Page.docx)

I tried to create the page....Not sure if it worked! Here's the text that I think we might want to add. I propose moving text from the apply page here to shorten the apply page and move this text where i think it might fit best.

**Comment by kategarklavs** [comment]
@kmonterr Howdy! I just reviewed the intro, and it looks good! And I hear you re: the About page being pretty content rich.

On the topic of the program's selling points — We'd included those in About as a means of (hopefully) converting potential applicants. Our goal for the About page was not only to provide necessary information about the program and its goals, but also to pitch the program (and its selling points) to potential applicants, increasing conversions. I'd recommend keeping the selling-point-related info on About to maintain the continuity of the narrative here.

File: pages/For Applicants.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+title: For applicants page
+layout: secondary
```
Comment (line47) [inline pages/For Applicants.md]: I would suggest using the `secondary-narrow` layout here. More info on layouts in the [Wiki](https://github.com/18F/nsf-sbir/wiki/Website:-Page-layouts).

**Comment by kmonterr** [comment]
@line47

**Pull request merged**